libs/server/repository/infrastructure/src/lib/bitbucket/repositories/bitbucket-repository.repository.ts

```diff
@@ -34,7 +34,12 @@ export class BitbucketRepositoryRepository extends RemoteRepositoryRepository {
         headers: { Authorization: `Bearer ${token}` }
       })
       .pipe(
-        map((res: AxiosResponse) => res.data),
+        map((res: AxiosResponse) => {
+          if (!(res.data.username || res.data.nickname)) {
+            throw new Error('No username or nickname provided');
```

Comment:
This must be something more specific than just an error. We need to handle this somehow in ErrorsInterceptor